randovania/games/prime1/exporter/scan_tips.py

```diff
@@ -0,0 +1,696 @@
+            "lock onto a Puffer to the right or left, and perform a lock-on dash towards the ledge."
+        ),
+    ],
+    "1839087853": [
+        # Tree Weakness Tip
+        # Replaces Main Plaza tree scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;Opening the tree requires a &main-color=#FF3333;Super Missile&main-color=#89D6FF;."
+            "\n\n\nAdditionally, it is possible to peek the item inside the tree without opening it. "
+            "To do so, stand at a specific angle and location on the right side of the tree's lip, then jump in place."
+        ),
+    ],
+    "1530752134": [
+        # Phendrana Canyon L-Jump Tip
+        # Replaces Phendrana Canyon platform scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;The item in this room can be acquired without the Scan Visor "
+            "by using the &main-color=#FF3333;Space Jump Boots&main-color=#89D6FF; and L-Jumping."
+        ),
+    ],
+    "3740523398": [
+        # Phendrana Canyon Escape Tip
+        # Replaces Phendrana Canyon half-pipe fish scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;As long as the boxes in this room remain intact, "
+            "they can be climbed to exit this room without any items."
+        ),
+    ],
+    "442524052": [
+        # Rapid Fire Missiles Tip
+        # Replaces Main Plaza holobanner
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;Hold&image=5D9F9796;and rapidly tap "
+            "&image=EA2A1C5C; to fire missiles as fast as possible."
+        ),
+    ],
+    "807982837": [
+        # Underwater Slope Jump Tip
+        # Replaces Watery Hall plant flavor text
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;Underwater slope jumps performed without the Gravity Suit allow access "
+            "to great heights.\n\nYou can use an underwater slope jump to reach the sunken item in this "
+            "room without the Gravity Suit. This is significantly easier with the "
+            "&main-color=#FF3333;Space Jump Boots&main-color=#89D6FF;."
+        ),
+    ],
+    "2077227745": [
+        # Meta Ridley Boost Tip
+        # Replaces Meta Ridley scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "Meta Ridley",
+        (
+            "&main-color=#89D6FF;During the first phase of combat against Meta Ridley, "
+            "hitting it with a &main-color=#FF3333;Boost Ball&main-color=#89D6FF; "
+            "charge causes enough damage to immediately move to the fight's second phase."
+        ),
+    ],
+    "2967215000": [
+        # Non-Superheated Back Magmoor Tip
+        # Replaces Puddle Spore scan
```

Comment:
I think this is good information to convey but choosing a good scan node is difficult. While I believe the puddle spore is not a good spot for it since it is not something players are likely to scan, there are not many options. Maybe the Magmoor Workstation pump monitor?

Reply:
I'm definitely open to moving it (I also think the Puddle Spores aren't great), but I think the Magmoor Workstation pump monitor text changes as you activate it - that, plus the fact that it's a functional hint for players, has me wary about using it instead.